cime_config/testdefs/testmods_dirs/clm/Fates/shell_commands

```diff
@@ -1,2 +1,16 @@
 ./xmlchange CLM_BLDNML_OPTS="-no-megan" --append
 ./xmlchange BFBFLAG="TRUE"
+
+CASE=`./xmlquery CASE --value`
+SRCROOT=`./xmlquery SRCROOT --value`
+
+#CIME_OUTPUT_ROOT=`./xmlquery CIME_OUTPUT_ROOT --value`
+# CASEDIR=`./xmlquery CASEROOT --value`
+# env_test.xml IS NOT AVAILABLE TESTROOT=`./xmlquery TESTROOT --value`
+# env_test.xml IS NOT AVAILABLE TESTID=`./xmlquery TESTID --value`
+# env_test.xml IS NOT AVAILABLE TESTARGV=`./xmlquery TEST_ARGV --value`
+
+FATESDIR=$SRCROOT/src/fates
+FATESPARAMFILE=${SRCROOT}/src/fates/parameter_files/binaries/${CASE}-params.nc
```

Comment:
These two lines are duplicated in various `shell_commands` files. Could they be moved to a new shell script that they all use? Maybe in a new directory `../_scripts/`.

Reply:
@samsrabin did this, so this is resolved.